seed/data_importer/match.py

```diff
@@ -256,7 +295,8 @@ def inclusive_match_and_merge(unmatched_state_ids, org, StateClass):
     # Collapse groups of matches found in the previous step into 1 -State per group
     merges_within_file = 0
     priorities = Column.retrieve_priorities(org)
-    for ids in matched_id_groups:
+    batch_size = int(len(matched_id_groups) / 100) + (len(matched_id_groups) % 100 > 0)
```

Comment:
`(len(matched_id_groups) % 100 > 0)` is a simple way of rounding up without importing a library. This avoid a Zero division error